cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -465,7 +478,7 @@ mkConsensusTracers' elidingFetchDecision measureBlockForging
   tracingVerbosity :: TracingVerbosity
   tracingVerbosity = traceVerbosity traceConf
 
-mkConsensusTracers' _ _ _ _ TracingOff _ = Consensus.Tracers
+mkConsensusTracers' _ _ _ _ _ TracingOff _ = Consensus.Tracers
```

Comment:
Surely there is a less horrible way to do that?

Reply:
That would take a refactor, and people complain about large changes -- making them hard to review.

Reply:
Suggestion: fix it in a separate commit (before the other changes)?

Reply:
@mrBliss, done!

I didn't clean up all things I could, but that's already a large change..